### DISCUSSION TITLE
Start on view: URL variable for view

### DISCUSSION THREAD

**Discussion by OlofBailey** [matt8707/ha-fusion#580] [Q&A]
Is there a way to add the view ID to the URL? 

I'm looking to use fusion to replace a bunch of touch panels around the house and want them to startup with their local rooms control views?

**Comment by esseki** [comment]
Hi, it is indeed possible. 
Check the doc here : https://github.com/matt8707/ha-fusion?tab=readme-ov-file#query-strings
